features/aanschrijfwijze/adellijk.feature

```diff
@@ -0,0 +1,363 @@
+      | naam                                 | waarde        |
+      | voornamen (02.10)                    | James Antoine |
+      | adellijke titel of predicaat (02.20) | JH            |
+      | voorvoegsel (02.30)                  | van den       |
+      | geslachtsnaam (02.40)                | Aedel         |
+      | aanduiding naamgebruik (61.10)       | <naamgebruik> |
+      En de persoon heeft een 'partner' met de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | voorvoegsel (02.30)   | de     |
+      | geslachtsnaam (02.40) | Boer   |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000139                       |
+      | fields              | adressering.aanschrijfwijze     |
+      Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
+      | naam                          | waarde                    |
+      | aanschrijfwijze.naam          | <naam in aanschrijfwijze> |
+      | aanschrijfwijze.aanspreekvorm | De hoogwelgeboren heer    |
+
+      Voorbeelden:
+      | naamgebruik | naam in aanschrijfwijze             |
+      | E           | Jonkheer J.A. van den Aedel         |
+      | V           | Jonkheer J.A. de Boer-van den Aedel |
+      | N           | Jonkheer J.A. van den Aedel-de Boer |
+
+    Scenario: Jonkheer en naamgebruik "P" voert geen predicaat en geen aanspreekvorm
+      Gegeven de persoon met burgerservicenummer '000000139' heeft de volgende gegevens
+      | naam                        | waarde |
+      | geslachtsaanduiding (04.10) | M      |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                                 | waarde        |
+      | voornamen (02.10)                    | James Antoine |
+      | adellijke titel of predicaat (02.20) | JH            |
+      | voorvoegsel (02.30)                  | van den       |
+      | geslachtsnaam (02.40)                | Aedel         |
+      | aanduiding naamgebruik (61.10)       | P             |
+      En de persoon heeft een 'partner' met de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | voorvoegsel (02.30)   | de     |
+      | geslachtsnaam (02.40) | Boer   |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000139                       |
+      | fields              | adressering.aanschrijfwijze     |
+      Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
+      | naam                          | waarde                 |
+      | aanschrijfwijze.naam          | J.A. de Boer           |
+      | aanschrijfwijze.aanspreekvorm | De hoogwelgeboren heer |
```

Comment:
Wat klopt hier niet? 'geen aanspreekvorm' in scenario titel of aanspreekvorm in dan stap?